src/pages/InformationSheetContent/InformationSheetContent.js

```diff
@@ -73,7 +73,7 @@ const InformationSheetContent = ({ crop, modalData }) => {
                 <AccordionDetails>
                   {' '}
                   <div className="row col-12 text-left">
-                    {cat.attributes.map((att) => (
+                    {cat.attributes.map((att) => (att.label !== 'Comments' ? (
```

Comment:
this doesn't handle notes for MCCC and NECCC. you should either handle those cases as well or just use the last index of the array as notes

Reply:
The last index wont work because not every category has a notes. only the midwest and south do so far so all the NE last item would be weird.  I will either have to target the name like "comments" or "Notes:" or i will need a new attribute added to the Notes or Comments object to signify when there is a note or attribute.  Once the notes section for the south is updated i will add a conditional for it too.

Reply:
good point. the NECCC and SCCC notes section should now be at the end of the array if that's what you're waiting on